portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,51 +30,76 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  final int SC_NO_CONTENT = 204;
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    for (Entity entity : results.asIterable()) {
+      datastore.delete(entity.getKey());
+    }
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Greeting");
+    Query query = new Query("Comment");
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<String> greetings = new ArrayList<>();
+    List<String> comments = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
-      String greeting = (String) entity.getProperty("greeting");
-      greetings.add(greeting);
+      String comment = (String) entity.getProperty("comment");
+
+      String language = request.getParameter("languages");
+
+      if (language != null && !language.equals("original")) {
+        Translate translate = TranslateOptions.getDefaultInstance().getService();
+        Translation translation = translate.translate(comment, Translate.TranslateOption.targetLanguage(language));
+        comment = translation.getTranslatedText();
+      }
+
+      comments.add(comment);
     }
     
-    response.setContentType("text/html;");
-    response.getWriter().println(convertGreetingsToJson(greetings));
+    response.setContentType("text/html");
```

Comment:
Why do you have content type html if you return json? You can use application/json